composer.json

```diff
@@ -1,10 +1,28 @@
 {
-    "require-dev": {
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
-        "wimg/php-compatibility": "dev-master",
-        "wp-coding-standards/wpcs": "dev-develop"
-    },
-    "scripts": {
-        "lint": "phpcs"
-    }
+  "name": "wordpress/gutenberg",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0",
+  "description": "Prototyping since 1440. Development hub for the editor focus in core.",
+  "homepage": "https://wordpress.github.io/gutenberg/",
+  "keywords": [
+    "gutenberg", "wordpress", "editor", "wp", "react", "javascript"
+  ],
+  "support": {
+    "issues": "https://github.com/WordPress/gutenberg/issues"
+  },
+  "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
+    "squizlabs/php_codesniffer": "2.9.x",
+    "wimg/php-compatibility": "dev-master",
+    "wp-coding-standards/wpcs": "^0.11.0"
```

Comment:
The latest released version of WPCS is `0.14.0`, so ideally this would be the minimum version here.

Reply:
Fixed by 4e4055f41a1611959eb6dabfd527985c7ebcd8a6